src/start_server.ts

```diff
@@ -10,11 +10,12 @@
 
 'use strict';
 
-const express = require('express');
+import * as express from 'express';
+import * as http from 'http';
+import { makeApp } from './make_app';
+import * as opn from 'opn';
+
 const findPort = require('find-port');
```

Comment:
not working... that style of export isn't a valid ES6 module, are you sure it's possible?